contracts/script/HelloWorldDeployer.s.sol

```diff
@@ -4,8 +4,14 @@ pragma solidity ^0.8.0;
 import {Script} from "forge-std/Script.sol";
 import {console2} from "forge-std/Test.sol";
 import {HelloWorldDeploymentLib} from "./utils/HelloWorldDeploymentLib.sol";
-import {CoreDeploymentLib} from "./utils/CoreDeploymentLib.sol";
+import {CoreDeploymentLib, StrategyManager} from "./utils/CoreDeploymentLib.sol";
```

Comment:
Can you import the StrategyManager separately like the StrategyFactory

Reply:
fixed!